binaries/apiserver/main.go

```diff
@@ -28,17 +29,22 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Start a new goroutine for bundlestore server as well as the observability server.
-	bag, schema := bundlestore.Defaults()
+	bag := ice.NewMagicBag()
+	schema := jsonconfig.EmptySchema()
+	bag.InstallModule(temp.Module())
```

Comment:
I think it'll go too stale, ultimately, if you want to see what's in a module, it's easier to go look at it than try and comment on every use.